src/styles/main.scss

```diff
@@ -0,0 +1,8 @@
+@import "./utils/variables.scss";
+@import "./utils/functions.scss";
+@import "./utils/mixins.scss";
+@import "./utils/animations.scss";
+
```

Comment:
Consider renaming `animations.scss` to `_animations.scss` to follow the partial naming convention in SCSS.